pkg/config/config.go

```diff
@@ -59,8 +57,6 @@ type Config struct {
 	client client.Client
 	// Default config selector
 	defaultConfigSelector *metav1.LabelSelector
-	// Default config for email, wechat, slack etc.
```

Comment:
The `type Config struct` could be renamed to `type Controller struct` to be more precise maybe?